apps/system/utils/utils_cpuload.c

```diff
@@ -249,7 +249,10 @@ static void cpuload_print_normal(void)
 				printf("  CPU%d |", j);
 			}
 		}
+#else
+		printf("  CPU  |");
 #endif
+		printf(" Task Name  ");
```

Comment:
Line 210,
```
#if (CONFIG_TASK_NAME_SIZE > 0)
	printf(" %s", stat_info[PROC_STAT_NAME]);
#endif
	printf("\n");
```
When the config is not enabled, it does not print thread name, but title prints it always.
How about modifying line 210 like below?
```
#if (CONFIG_TASK_NAME_SIZE > 0)
	printf(" %s\n", stat_info[PROC_STAT_NAME]);
#else
	printf(" NA\n"); 
#endif
```
Need to remove line 213